release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,187 @@
+k6 `v1.0.0-rc1` is here 🎉! This release includes:
+
+- A preview of the next `v1.0.0` release
+- `k6/experimental/webcrypto` promoted to stable and available globally under `crypto`.
+- A revamped end-of-test summary aiming to bring an easier way to understand test results.
+- `k6/browser` provided an API for tracking network requests and responses.
+
+## Breaking changes
+
+- [#4541](https://github.com/grafana/k6/pull/4541) Commas(`,`) are now supported in the values of `--tag` cli flag. This is a breaking change as previously a comma meant the start of a new set of tag-values. As comma is a valid symbol for the value of a tag this is necessary to have equivalency between different ways of setting tags. This still allows multiple tags to be set on the cli with multiple `--tag key=value` arguments.
+
+### A revamped end-of-test summary aiming to bring an easier way to understand test results [#4089](https://github.com/grafana/k6/pull/4089)
```

Comment:
This doesn't mention that the --summary-export is also changed by default

Reply:
@joanlopez can you address it, please?

Reply:
Waiting on https://github.com/grafana/k6/pull/4647 to decide what to write exactly.